cloud/passportauth/src/auth/DefaultStrategy.ts

```diff
@@ -1,7 +1,8 @@
-import { BunyanLogger, Logger } from '@raincatcher/logger';
```

Comment:
Aways make sure that you pushing changes that are part of the ticket.  Let's remove this change.